hack/local-up-cluster.sh

```diff
@@ -410,7 +410,9 @@ EOF
 
     # Wait for kube-apiserver to come up before launching the rest of the components.
     echo "Waiting for apiserver to come up"
-    kube::util::wait_for_url "https://${API_HOST}:${API_SECURE_PORT}/version" "apiserver: " 1 ${WAIT_FOR_URL_API_SERVER} || exit 1
+    # this uses the API port because if you don't have any authenticator, you can't seem to use the secure port at all.
```

Comment:
Is this related (vs just something you found while fixing and are fixing in this PR).  I don't have enough understanding on this.